platform-docs/docs/create-block/attributes.md

```diff
@@ -48,27 +48,27 @@ registerBlockType( 'gutenpride/gutenpride-block', {
 
 ## TextControl Component
 
-For our example block, the component we are going to use is the **TextControl** component, which is similar to an HTML text input field. You can see [the documentation for the `TextControl` component](https://developer.wordpress.org/block-editor/reference-guides/components/text-control/). You can browse an [interactive set of components in this Storybook](https://wordpress.github.io/gutenberg/).
+For our example block, the component we are going to use is the **PlainText** component, which allows the user to type some non formatted text. The **PlainText** component is imported from the `@wordpress/block-editor` package.
```

Comment:
Tiny nit: feel free to ignore! (unformatted instead of non formatted)

```suggestion
For our example block, the component we are going to use is the **PlainText** component, which allows the user to type some unformatted text. The **PlainText** component is imported from the `@wordpress/block-editor` package.
```